python/pipeline/odor.py

```diff
@@ -0,0 +1,268 @@
+import h5py
+import os
+
+import datajoint as dj
+import numpy as np
+
+from pipeline import mice
+from pipeline import meso
+from pipeline.utils import h5
+from commons import lab
+
+schema = dj.schema('pipeline_odor', locals(), create_tables=False)
+CURRENT_VERSION = 1
```

Comment:
you are not using the version so you may not need this. and if you are, it's probably better to have a version (as secondary attribute) per table rather than one version as PK over the entire schema